nanomq/apps/broker.c

```diff
@@ -8,22 +8,25 @@
 //
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdarg.h>
+#include <syslog.h>
+#include <signal.h>
+#include <ctype.h>
+#include <unistd.h>
 
 #include <hash.h>
 #include <mqtt_db.h>
 #include <nng.h>
 #include <protocol/mqtt/mqtt_parser.h>
 #include <protocol/mqtt/nano_tcp.h>
 #include <zmalloc.h>
-
-#if (defined DEBUG) && (defined ASAN)
 #include <signal.h>
```

Comment:
signal.h has been include at line 13, fix it in next commit.

Reply:
Understood. I believe signal.h should have been included by default.